src/NPacMan.Game/GameSettingsLoader.cs

```diff
@@ -49,13 +55,13 @@ public static IGameSettings Load(string board)
                             pacMan = new PacMan(columnNumber - 1, rowNumber, Direction.Left, PacManStatus.Alive, 3);
                             break;
                         case 'X':
-                            walls.Add((columnNumber-1, rowNumber));
+                            walls.Add((columnNumber - 1, rowNumber));
                             break;
                         case '.':
```

Comment:
We can either include cases for ghost home positions, or simply add the home positions to the coin list

Reply:
Good point - should be sorted now.